src/components/insert_blog/InsertStyle.css

```diff
@@ -0,0 +1,9 @@
+html, body, #root {
+    width: 100%;
+    height: 100%;
+  }
+  
+  .App {
```

Comment:
Why is the CSS of the `App` inside this component? This CSS file should only contain the CSS of the `InsertBlog` component.